spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -385,6 +385,9 @@ def getRegistrationStackSh(saltEnabled):
         elif [ -f /etc/oracle-release ]; then
             grep -v '^#' /etc/oracle-release | grep -q '\(Oracle\)' && BASE="oracle"
             VERSION=`grep -v '^#' /etc/oracle-release | grep -Po '(?<=release )\d+'`
+        elif [ -f /etc/alinux-release ]; then
```

Comment:
Too bad I didn't merge Amazon Linux yet. For Amazon Linux it works just by parsing `/etc/os-release` without any regex :-)